packages/tailwindcss/src/css-parser.ts

```diff
@@ -286,10 +286,14 @@ export function parse(input: string) {
       }
 
       let declaration = parseDeclaration(buffer, colonIdx)
-      if (parent) {
-        parent.nodes.push(declaration)
+      if (declaration) {
+        if (parent) {
+          parent.nodes.push(declaration)
+        } else {
+          ast.push(declaration)
+        }
       } else {
-        ast.push(declaration)
+        throw new Error(`Invalid custom property, expected a value`)
```

Comment:
Now that we're throwing errors maybe we can eliminate some of the nesting here?

```css
let declaration = parseDeclaration(buffer, colonIdx)
if (!declaration) {
  throw new Error(`Invalid custom property, expected a value`)
}

if (parent) {
  parent.nodes.push(declaration)
} else {
  ast.push(declaration)
}
```

Reply:
Yeah I think that would be a good change, the indentation is already quite rough here 👍

Reply:
Nice, ok will update 👍